com.github.actionless.oomox.json

```diff
@@ -0,0 +1,126 @@
+                    "dest": "./deps/sassc/"
+                }
+            ]
+        },
+        {
+            "name": "imagemagick",
+            "config-opts": [
+                "--enable-static=no",
+                "--disable-docs",
+                "--disable-deprecated",
+                "--without-autotrace",
+                "--without-bzlib",
+                "--without-djvu",
+                "--without-dps",
+                "--without-fftw",
+                "--without-fontconfig",
+                "--without-fpx",
+                "--without-freetype",
+                "--without-gvc",
+                "--without-jbig",
+                "--without-jpeg",
+                "--without-lcms",
+                "--without-lzma",
+                "--without-magick-plus-plus",
+                "--without-openexr",
+                "--without-openjp2",
+                "--without-pango",
+                "--without-raqm",
+                "--without-tiff",
+                "--without-webp",
+                "--without-wmf",
+                "--without-x",
+                "--without-xml",
+                "--without-zlib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.imagemagick.org/download/ImageMagick-7.0.7-22.tar.xz",
+                    "sha256": "49de9e08ea255a1f939158d85d50dfa29285bccbdcb7fee0fe4309061d438489"
+                }
+            ]
+        },
+        {
+            "name": "oomox",
+            "buildsystem": "simple",
+            "build-commands": [
+                "rm -r ./plugins/{oomoxify,theme_materia}",
+                "sed -i -e 's/\\/opt\\/oomox\\//\\/app\\/opt\\/oomox\\//g' ./packaging/bin/*",
+                "./packaging/install.sh ./ /app/",
+                "python3 -O -m compileall /app/opt/oomox/oomox_gui",
+
+                "install --target-directory=/app/bin /app/usr/bin/*",
+                "rm -r /app/usr/bin/"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/actionless/oomox.git",
+                    "tag": "1.5.0.2"
+                }
+            ]
+        },
+        {
+          "name": "appdata",
+          "buildsystem": "simple",
+          "build-commands": [
+            "mkdir -p /app/share/appdata",
+            "install -Dp -m 644 com.github.actionless.oomox.appdata.xml /app/share/appdata",
+            "install -Dp -m 644 com.github.actionless.oomox-64.png /app/share/icons/hicolor/64x64/apps/com.github.actionless.oomox.png",
+            "install -Dp -m 644 com.github.actionless.oomox-128.png /app/share/icons/hicolor/128x128/apps/com.github.actionless.oomox.png",
+            "install -Dp -m 644 com.github.actionless.oomox.png /app/share/icons/hicolor/scalable/apps/com.github.actionless.oomox.png",
```

Comment:
I've just noticed, this is a PNG. You should only put SVGs in the scalable directory. Otherwise if you just want to ship a big icon for resizing, put in the an appropriate directory, you can make it as big as you like!